app/templates/view_users.html

```diff
@@ -113,7 +122,7 @@ <h1 class="govuk-heading-xl">Find a user</h1>
 
       {% set row = ns.rows.append(
         [
-          {"text": user.name},
+          {"text": name},
```

Comment:
I would need to test, but in the case name is a link, shouldn't we use "html" instead of "text"?

Reply:
Hmm. That's odd. I agree with you - I would have thought so as well, but 'text' seems to work fine - the link renders correctly. If I switch it over to 'html', I can see no difference. :shrug: 

But there doesn't seem to be any harm, and 'html' is correct, so I've switched over